package.json

```
@@ -172,6 +173,7 @@
 		"@fluidframework/test-tools": "^1.0.195075",
 		"@microsoft/api-documenter": "^7.21.6",
 		"@microsoft/api-extractor": "7.50.1",
+		"@vvago/vale": "^3.10.0",
```

Because it exists, but really just to get its transitive dependency on axios bumped to 1.8.x. It's currently resolving to 1.7.9 which is flagged by component governance.
```suggestion
		"@vvago/vale": "^3.11.2",
```